soc/v2/nios2f-zephyr/Kconfig.soc

```
@@ -1,7 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_NIOS2F_ZEPHYR
-	bool "Nios IIf - Zephyr Golden Configuration"
+	bool
```

there should be a `Kconfig.soc` with the general definition which can be reused in other Kconfig trees, such as sysbuild.
```
config SOC_NIOS2F_ZEPHYR
	bool
```

so that this file just have:
```suggestion
config SOC_NIOS2F_ZEPHYR
```